components/AppHeader.vue

```diff
@@ -80,6 +80,13 @@
           </Button>
         </div>
       </nav>
+      <Label
+        :v-if="IS_TESTNET"
```

Comment:
Are you sure IS_TESTNET is not undefined?

Reply:
It's a string 'TRUE' in the test environment